src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -246,6 +246,33 @@ pipelines:
           filename: upgrade-3.1.0.19-to-3.1.0.20.sql
           updateIntegrity: true
 
+    - currentVersion: 3.1.0.20
+      nextVersion: 3.1.0.21
+      operations:
+        - type: dbScriptUpgrader
+          filename: upgrade-3.1.0.20-to-3.1.0.21.sql
+          updateIntegrity: true
+        - type: blueprintRenameUpgrader
+          blueprintFolderName: website_editorial
+          newFolderName: 1000_website_editorial
+          commitDetails: Rename Editorial blueprint
```

Comment:
The `commitDetails` property will be ignored if we don't use the inherited method for doing the commit